js/sdk/src/globals.ts

```diff
@@ -0,0 +1,41 @@
+import { GlobalInternalSymbol } from './v2/utils'
+
+type ActiveRequest = {
+  serviceId: string
+  payload?: unknown
+  time: Date
+}
+
+type ActiveRequestGlobals = {
+  all: () => ActiveRequest[]
+  [GlobalInternalSymbol]: {
+    register: (sym: symbol, activeRequest: ActiveRequest) => unknown
+    unregister: (sym: symbol) => unknown
+  }
+}
+
+/**
+ * Global object for accessing currently active requests
```

Comment:
Regarding globals: it would make sense from an API standpoint to offer the `activeRequests` from each SDK instance. A global variable is only needed to alert the user when they creates multiple SDK instances, which is almost always a mistake.

Reply:
My initial thought is that with `globals` the user will be able to write it in the global/static scope.
I often see this pattern for telemetry purposes, which is written before the main function.

As with multiple SDKs, I think we should go with `console.warn`-ing instead when 2 or more instances are created.

Reply:
okay, makes sense